### PR TITLE
bazel: add aspect rules_ts,jest

### PR DESCRIPTION
**Pull request by jbedard**

### CODE REVIEW

File: WORKSPACE [hunk 1]
```diff
@@ -20,16 +20,30 @@ http_archive(
...
+    url = "https://github.com/aspect-build/rules_ts/archive/refs/tags/v1.0.2.tar.gz",
+)
+
+http_archive(
+    name = "aspect_rules_jest",
+    sha256 = "dd596891aa893048d2e8d210fce214459df33d454bf0e77906ebbfaee38f2bbc",
+    strip_prefix = "rules_jest-0.12.1",
+    url = "https://github.com/aspect-build/rules_jest/archive/refs/tags/v0.12.1.tar.gz",
+)
+
```
Comment (kormide) [inline WORKSPACE:42]: Are there any jest macros we need to load here?

Comment (jbedard) [reply WORKSPACE:42]: Good call, updated 👍

**Review by kormide — APPROVED** [review]

File: WORKSPACE [hunk 2]
```diff
@@ -20,23 +20,40 @@ http_archive(
...
+# Node toolchain setup ==========================
 load("@rules_nodejs//nodejs:repositories.bzl", "DEFAULT_NODE_VERSION", "nodejs_register_toolchains")
 
 nodejs_register_toolchains(
     name = "nodejs",
     node_version = DEFAULT_NODE_VERSION,
 )
 
+# rules_js setup ================================
```
Comment (kormide) [inline WORKSPACE:51]: Not a big fan of these headers 🤷‍♂️

Comment (jbedard) [reply WORKSPACE:51]: You don't like my beautiful art? :(

I just find the WORKSPACE becomes such a mess I like trying to separate things. Up to you though, I'll delete them?

**Pull request merged**